source/images.py

```diff
@@ -0,0 +1,129 @@
+from typing import List
+
+import numpy as np
+from PIL import Image
```

Comment:
Since the class should be renamed we might as well be explicit about this.
```suggestion
import PIL
```